core/iwasm/aot/aot_runtime.c

```diff
@@ -3125,8 +3125,32 @@ aot_alloc_frame(WASMExecEnv *exec_env, uint32 func_index)
     frame->func_perf_prof_info = func_perf_prof;
 #endif
     frame->sp = frame->lp + max_local_cell_num;
+
 #if WASM_ENABLE_GC != 0
     frame->frame_ref = (uint8 *)(frame->sp + max_stack_cell_num);
+
+    /* Initialize frame ref flags for import function */
+    if (func_index < module->import_func_count) {
+        AOTFuncType *func_type = module->import_funcs[func_index].func_type;
+        uint8 *frame_ref = frame->frame_ref;
+        uint32 i, j, k, value_type_cell_num;
+
+        for (i = 0, j = 0; i < func_type->param_count; i++) {
+            if (wasm_is_type_reftype(func_type->types[i])
+                && !wasm_is_reftype_i31ref(func_type->types[i])) {
+                frame_ref[j++] = 1;
+#if UINTPTR_MAX == UINT64_MAX
+                frame_ref[j++] = 1;
+#endif
+            }
+            else {
+                value_type_cell_num =
+                    wasm_value_type_cell_num(func_type->types[i]);
+                for (k = 0; k < value_type_cell_num; k++)
+                    frame_ref[j++] = 0;
+            }
+        }
```

Comment:
if func_type->param_count == 0, then the reserved 2 slots in frame_ref may remain uninitialized. Seems we should clear the frame_ref[0] and frame_ref[1] in this case:

```suggestion
        }
        
        if (func_type->param_count == 0) {
            frame_ref[0] = frame_ref[1] = 0;
        }
```

Reply:
Done.